src/npctalk.cpp

```diff
@@ -85,6 +85,18 @@ struct talk_trial {
     talk_trial( JsonObject );
 };
 
+struct talk_topic {
```

Comment:
Somewhat verbose. Could the constructor not be on a single line and the comment be reduced to a single line?